camera-config/cameras.rules

```diff
@@ -3,9 +3,14 @@
 #
 # To help write these rules, try running:
 #   udevadm info -a --name /dev/video0
+#
+# Install this in /etc/udev/rules.d
 
 # Mast camera
-SUBSYSTEM=="video4linux", ATTRS{idVendor}=="05a3", ATTRS{idProduct}=="9601", SYMLINK+="video20"
+SUBSYSTEM=="video4linux", ATTRS{idVendor}=="05a3", ATTRS{idProduct}=="9601", KERNELS=="1-2.3", SYMLINK+="video20"
 
 # Forearm camera
-SUBSYSTEM=="video4linux", ATTRS{idVendor}=="05a3", ATTRS{idProduct}=="9230", SYMLINK+="video30"
+SUBSYSTEM=="video4linux", ATTRS{idVendor}=="05a3", ATTRS{idProduct}=="9601", KERNELS=="1-2.1", SYMLINK+="video30"
```

Comment:
This means that it's now important that these cameras get plugged into the same USB ports every time, right?

Reply:
Right. As far as I could tell, there was no other way to distinguish between the forearm cam and the mast cam.

Actually, I still didn't manage to get all three cameras working simultaneously. I think the udev rules worked, but `motion` was still having trouble. Here's hoping I'll be able to find a good solution tomorrow morning at 8:00....